daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ScriptF.scala

```diff
@@ -441,11 +441,18 @@ object ScriptF {
       key: AnyContractKey,
   ) extends Cmd {
     private def translateKey(
-        env: Env
+        env: Env,
+        enableContractUpgrading: Boolean,
```

Comment:
We plan to drop this flag. Upgradability will need to be determined by language version. (fyi)